float64[] applied_effort
int8[]    check_contact
float64 timestamp
